Lesson4HomeWork.playground/Contents.swift

```diff
@@ -0,0 +1,21 @@
+import UIKit
+
+var arrayNumbers: [Int] = [-1, 2, 6, 12, 0, 5, 7, 8]
+
+for elements in 0...arrayNumbers.count - 1 {
+    let powNumbers = Int(pow(Double(arrayNumbers[elements]), Double(2)))
```

Comment:
There's no need to cast `2` to `Double` explicitly, you can just leave `2` here and the compilator will treat it as `Double` type